docs/Test-Strategy-Info.mdx

```diff
@@ -1,31 +1,182 @@
-* It could be good to have either one or multiple user journeys that navigates through all page types (e.g front page → MAP → another MAP → front page → article etc.). This simulate a real user experience and could also serve as a sanity check as mentioned above. This is currently limited by some page types still being on PAL, but there are still some user journeys currently possible without needing to touch PAL pages. Another issue is that some page types require an override on the URL to be used in the test environment as they do not have many/any assets on test (e.g On Demand Radio brands and episodes)
-* Layout changes at different viewport to be considered adding to cypress tests.
-* Cypress tests to be run on a subset of services (services having script switcher, RTL, different layout) and not for all services.
+While this document focuses primarily on E2E test strategy, it touches on other levels of testing to give a holistic understanding and context for the decisions we make when it comes to testing. It documents some of our existing approaches, while suggesting gradual improvements for the future. It’s a living collaborative document that we should refer to and update continuously. 
+
+
+
+
+## Why should we automate?
+
+There are many reasons why we write automated tests. Some of the most common are:
+
+
+1. Verify the code is working correctly
+2. Prevent future regressions
+3. Document code behaviour
+4. Provides (code) design guidance
+5. Support refactoring
+
+[source: https://madeintandem.com/blog/five-factor-testing/]
+
+
+> 🔊 Discussion point with team:
+> - Any other reasons? 
+> - What are for you the most important reasons?
+    
+
+It is important to think about why we write tests because it influences what types of testing we should prioritise. For example, unit tests might be good for improving the system design as the units become more independent and easier to test, but having only unit tests doesn’t necessarily verify the code is working correctly (beyond the single unit) as it won’t necessarily reveal problems that occur in the integration of components. On the other hand, E2E tests can provide better verification of the software, but they’re expensive and would not necessarily improve software design, nor document the code behaviour at unit or component level.
+
+## Where should we automate?
+### Defining the test levels
+
+Broadly speaking, we follow the test pyramid. We have a large set of unit tests, a smaller subset of integration tests, and an even smaller subset of UI tests. We should continue to follow this pattern.
+
+![Image](https://blog.getmason.io/content/images/2020/11/Testing-pyramid--6--1.jpg)
+Image [Source](https://blog.getmason.io/content/images/2020/11/Testing-pyramid--6--1.jpg)
+To avoid confusion, we will clarify what the test levels mean in our context.
```

Comment:
Add whitespace to put text on new line

```suggestion
Image [Source](https://blog.getmason.io/content/images/2020/11/Testing-pyramid--6--1.jpg)

To avoid confusion, we will clarify what the test levels mean in our context.
```